### PR TITLE
create JDK8-based WebClient implementation

### PR DESCRIPTION
**Pull request by russgold**
Implemented WebClient using HttpURLConnection, so that it is usable in JDK8, where most Weblogic images will presumably run under the operator.

Fixed the erroneous swapping of the Apache-based and JDK-based implementation names.

### CODE REVIEW

File: wls-exporter-core/src/main/java/com/oracle/wls/exporter/WebClient8Impl.java
Comment (ddsharpe) [inline wls-exporter-core/src/main/java/com/oracle/wls/exporter/WebClient8Impl.java:19]: Javadoc please

**Review by ddsharpe — APPROVED** [review]

File: wls-exporter-core/src/main/java/com/oracle/wls/exporter/WebClient8Impl.java
Comment (rjeberhard) [inline wls-exporter-core/src/main/java/com/oracle/wls/exporter/WebClient8Impl.java:1]: // Copyright (c) 2020, Oracle Corporation and/or its affiliates.
// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.

Comment (rjeberhard) [reply wls-exporter-core/src/main/java/com/oracle/wls/exporter/WebClient8Impl.java:1]: Missing "(c)" and comma

File: wls-exporter-core/src/test/java/com/oracle/wls/exporter/WebClient8ImplTest.java
Comment (rjeberhard) [inline wls-exporter-core/src/test/java/com/oracle/wls/exporter/WebClient8ImplTest.java:1]: Add copyright

**Review by rjeberhard — APPROVED** [review]
Approving with the expectation that the two copyrights will be fixed before merging.

**Pull request merged**
